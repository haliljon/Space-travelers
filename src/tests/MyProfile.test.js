import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyProfile from '../components/myProfile';
import store from '../redux/configureStore';

it('matches snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
