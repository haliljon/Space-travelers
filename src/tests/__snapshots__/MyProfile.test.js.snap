// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row justify-content-evenly"
  >
    <div
      className="col-6"
    >
      <h1
        className="p-2"
      >
        My Missions
      </h1>
      <div>
        No Missions Joined
      </div>
      <div />
    </div>
    <div
      className="col-6"
    >
      <h1
        className="p-2"
      >
        My Rockets
      </h1>
      <div>
        No Rockets Reserved
      </div>
      <div />
    </div>
    <div
      className="col-6 mt-5"
    >
      <h1
        className="p-2"
      >
        My Dragons
      </h1>
      <div>
        No Dragons Booked
      </div>
      <div />
    </div>
  </div>
</div>
`;
